src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -122,18 +122,20 @@ public CaseResponse transformExceptionRecord(ExceptionRecord exceptionRecord, bo
             return formTypeValidatorResponse;
         }
 
-        String formType = exceptionRecord.getFormType();
-        log.info("formtype for case {} is {}", caseId, formType);
 
-        if (formType == null || notAValidFormType(formType)) {
-            ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
-            String ocrFormType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+        ScannedData scannedData = sscsJsonExtractor.extractJson(exceptionRecord);
+        String formType = getField(scannedData.getOcrCaseData(), FORM_TYPE);
+
+        if (formType == null  || notAValidFormType(formType)) {
+            formType = exceptionRecord.getFormType();
 
-            if (!notAValidFormType(ocrFormType)) {
-                formType = ocrFormType;
+            if (formType == null || notAValidFormType(formType)) {
+                formType = null;
```

Comment:
Your unit tests are not covering this line, can you add tests for it?

Reply:
same as above

Reply:
![image](https://user-images.githubusercontent.com/18538046/185188358-34feb09c-2237-49d5-af30-bf28343fe04e.png)
Still need tests to cover this